context_response_test.go

```diff
@@ -299,6 +301,57 @@ func TestResponse(t *testing.T) {
 			expectCode: http.StatusOK,
 			expectBody: "Goravel",
 		},
+		{
+			name:       "WithoutCookie",
+			method:     "GET",
+			url:        "/without/cookie",
+			cookieName: "goravel",
+			setup: func(method, url string) error {
+				fiber.Get("/without/cookie", func(ctx contractshttp.Context) contractshttp.Response {
+					return ctx.Response().WithoutCookie("goravel").String(http.StatusOK, "Goravel")
```

Comment:
S: Need to add cookie when making the request, the `WithoutCookie` method will be used.

Reply:
I am adding cookies here 
![image](https://github.com/goravel/fiber/assets/84431594/38bd01c0-a42e-4bcf-b7ab-93d5e199f8e4)